grpc/service_config/service_config.proto

```diff
@@ -236,11 +236,35 @@ message WeightedTargetLoadBalancingPolicyConfig {
   map<string, Target> targets = 1;
 }
 
+// Configuration for xds_cluster_manager_experimental LB policy.
+message XdsClusterManagerLoadBalancingPolicyConfig {
+  message Child {
+    repeated LoadBalancingConfig child_policy = 1;
```

Comment:
Should this `json_name` to `child_policy`?

Reply:
I don't think so.  We use the normal JSON form (`childPolicy`) for these fields in gRPC.